QuantumLink/Server/Networking/Session.cs

```diff
@@ -1,65 +1,65 @@
-﻿using QuantumLink.networking.protocol;
+﻿using QuantumLink.Common.Networking.Protocol;
 using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Net.Sockets;
 using System.Threading;
 
-namespace Server.networking
+namespace QuantumLink.Server.Networking
 {
     public delegate void ClientDisconnectedEventHandler(Session session);
 
     public class Session : IDisposable
     {
         protected delegate void OpcodeHandler(InboundPacket inboundPacket);
 
-        protected readonly TcpClient client;
-        protected readonly NetworkStream networkStream;
+        protected readonly TcpClient Client;
+        protected readonly NetworkStream NetworkStream;
 
-        protected readonly Thread listenThread;
-        private volatile bool stopping;
+        protected readonly Thread ListenThread;
+        private volatile bool _stopping;

```

Comment:
Whats up with the naming conventions. I think we should establish a unified naming convention for this project before moving onwards

Reply:
Yes, the microsoft one